src/Security/OpenIdConfigurationProvider.php

```diff
@@ -192,10 +197,32 @@ public function getAuthorizationUrl(array $options = []): string
      *
      * @throws ItkOpenIdConnectException
      */
-    public function validateIdToken(string $idToken, string $nonce): object
+
+    /**
+     * Do any required verification of the id token and return an array of decoded claims
+     *
+     * @param string $idToken
+     *   Raw id token as string
+     *
+     * @param string $nonce
+     *   Nonce as string
```

Comment:
Good catch on the missing docs parameter 👍 

(And again - Let's drop "as string")

Reply:
Fixed in 860c85a